Makefile

```diff
@@ -78,7 +78,7 @@ GOBIN ?= ${CURDIR}/bin
 KUSTOMIZE_VERSION ?= v5.3.0
 KUSTOMIZE ?= ${GOBIN}/kustomize
 
-CONTROLLER_GEN_VERSION ?= v0.8.0
+CONTROLLER_GEN_VERSION ?= v0.12.0
```

Comment:
`creationTimestamp: null` was too old, and parsing failed when applying the manifests. This is an improvement.